src/message.h

```diff
@@ -2517,7 +2517,7 @@ static  Q8tkWidget *MAKE_KEY_COMBO(Q8tkWidget *box,
 
     q8tk_combo_set_text(combo, initstr);
     q8tk_misc_set_size(combo, 6, 0);
-    q8tk_signal_connect(combo, "activate", callback, (void*)(p->val));
+    q8tk_signal_connect(combo, "activate", callback, (void*)((intptr_t)p->val));
```

Comment:
```suggestion
    q8tk_signal_connect(combo, "activate", callback, (void *)((intptr_t)p->val));
```
For consistent spacing.